### DISCUSSION TITLE
Unable to map Push event payload mapping to the PR that caused it

### DISCUSSION THREAD

**Discussion by AaronFriel** [community/community#17344] [Actions]
On a push event, a GitHub Action may want to update the PR with additional data or comments, such as the result.

- https://github.com/pulumi/actions/issues/621

I was unable to find in the docs how one might be able to reliably - without heuristics - obtain the PR number that resulted in a commit being pushed. Without that, we'd be worried that attempting to use heuristics to implement the feature above reliably.

If the `push` event payload contained the PR number, if one was present, we would be able to make that link and update the PR with a comment or metadata with the outcome of the pull request merge.

**Comment by ghost** [comment]
> On a push event, a GitHub Action may want to update the PR with additional data or comments

Yes! Especially a push that’s a merge to the default branch.

I’ve needed this more than a few times.
